frontend/src/components/NavigationButton.vue

```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+import { useAttrs } from 'vue';
+
+const buttonAttrs = useAttrs();
+</script>
+
+<template>
+    <div class="navigation-button-container">
+        <button>
+            <span v-if="buttonAttrs['navigateTo'] === 'left'">&#171;</span>
```

Comment:
Hum, how about we use this interface ?
```typescript
interface NavigateButtonProps {
  backgroundColor: string;
  chevronLeft?: boolean;
  chevronRight?: boolean;
}
```

and your component would be 
```vue
<span v-if="props.chevronLeft">&#171;</span>
```

You can find another name if `chevronLeft` and `chevronRight` seems weird

Reply:
That seems way more better, thanks ! And `chevronLeft`/`chevronRight` are good to me